libusb/os/windows_winusb.c

```diff
@@ -525,7 +525,8 @@ static int windows_assign_endpoints(struct libusb_device_handle *dev_handle, uin
 
 	if (iface >= conf_desc->bNumInterfaces) {
 		usbi_err(HANDLE_CTX(dev_handle), "interface %d out of range for device", iface);
-		return LIBUSB_ERROR_NOT_FOUND;
+		r = LIBUSB_ERROR_NO_MEM;
```

Comment:
You change it from LIBUSB_ERROR_NOT_FOUND to LIBUSB_ERROR_NO_MEM, why?

Reply:
```suggestion
		r = LIBUSB_ERROR_NOT_FOUND;
```